src/sentry/sentry_apps/services/hook/impl.py

```diff
@@ -21,13 +21,61 @@ def update_webhook_and_events(
         with transaction.atomic(router.db_for_write(ServiceHook)):
             hooks = ServiceHook.objects.filter(application_id=application_id)
             if webhook_url:
-                for hook in hooks:
-                    hook.url = webhook_url
-                    hook.events = expand_events(events)
-                    hook.save()
-                return [serialize_service_hook(h) for h in hooks]
+                expanded_events = expand_events(events)
+                hooks.update(url=webhook_url, events=expanded_events)
```

Comment:
When testing this logic with 10000 servicehooks to update, the test finished in 7s compared to 6s for 1000 servicehooks (via tests). So I'm honestly not sure if in production how performant the query would be? Can I assume in those 7s the db also finished completing the op/query.

My hope that the operation would take >30s total in prod so the task won't timeout :oop .